disconnected/mirroring/about-installing-oc-mirror-v2.adoc

```diff
@@ -130,8 +130,21 @@ include::modules/oc-mirror-imageset-config-parameters-v2.adoc[leveloffset=+1]
 
 // Command reference for oc-mirror v2
 include::modules/oc-mirror-command-reference-v2.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
I don't think these additional resources are needed as the section exists in the same book. Also consider that ToolX changes might require us to remove such headings.